values.yaml

```diff
@@ -17,8 +21,8 @@ config:
   policy: {}
 
 authenticate:
-  # fullnameOverride: authenticate
-  # nameOverride: authenticate
+  fullnameOverride: authenticate
```

Comment:
Same comment as for `name/fullname`

Reply:
Yes, unfortunately, it is required when running `helm lint --strict`.

We can change it to `""` if you think it's better.

Reply:
See other comment on overrides.